en/components/grid-sorting.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Grid Sorting - Grid Feature
+_description: The Grid Sorting allows the ordering of records based on the values of a column.
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
+---
+
+## Grid Sorting
+
+Use Grid Sorting to indicate the ordering of records in ascending and descending order according to the values within a given column. The Grid Sorting is visually identical to the [Ignite UI for Angular Grid Sorting Feature](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/sorting.html)
+
+### Grid Sorting Demo
+
+<img class="responsive-img" src="../images/grid_sorting_demo.png" srcset="../images/grid_sorting_demo@2x.png 2x" />
+
+### Header Cell Feature
+
+In order to indicate a column as sortable use either the `Feature Left` or `Feature Right` overrides and set it to `Sorting`. Then you can set the `Sorting State` to Ascending, Descending or **Unsorted** to indicate the sort order. Of course, you would probably want to manipulate the column values to reflect the selected ordering.
```

Comment:
Only Unsorted setting is shown in bold. Unify bold setting for the three values included in list